mobile-app/app/screens/AppNavigator/screens/Balances/components/BalanceControlCard.tsx

```diff
@@ -0,0 +1,78 @@
+import { tailwind } from '@tailwind'
+import { ThemedIcon, ThemedText, ThemedView } from '@components/themed'
+import React from 'react'
+import { useWalletContext } from '@shared-contexts/WalletContext'
+import { View } from '@components'
+import { TouchableOpacity } from 'react-native'
+import { translate } from '@translations'
+import { useDeFiScanContext } from '@shared-contexts/DeFiScanContext'
+import { openURL } from '@api/linking'
+// @ts-expect-error
+import Avatar from 'react-native-boring-avatars'
+import { NavigationProp, useNavigation } from '@react-navigation/native'
+import { BalanceParamList } from '@screens/AppNavigator/screens/Balances/BalancesNavigator'
+import { IconButton } from '@components/IconButton'
+
+export function BalanceControlCard (): JSX.Element {
+  const { address } = useWalletContext()
+  const { getAddressUrl } = useDeFiScanContext()
+  const navigation = useNavigation<NavigationProp<BalanceParamList>>()
+  return (
+    <ThemedView
+      testID='balance_control_card'
+      style={tailwind('p-4 flex flex-col')}
+      dark={tailwind('bg-gray-800 border-b border-gray-700')}
+      light={tailwind('bg-white border-b border-gray-100')}
```

Comment:
![image](https://user-images.githubusercontent.com/40191153/140018134-8a260ad7-5a73-409d-ac4b-1d0d733fbc2d.png)
Can change the border color to a darker gray on light mode? Currently the border is same color as the background so can't distinguished, works fine on dark mode.

Reply:
Updated